action.js

```diff
@@ -126,31 +132,32 @@ module.exports.getBodyFromGitHubContext = getBodyFromGitHubContext;
 // the user wishes us to do a best-effort fill of the events body using all the
 // information available in the GitHub context object.
 //
-const constructBody = ( change_id,
-                        custom,
-                        github,
-                        pipeline_id,
-                        stage,
-                        stage_ref,
-                        status,
-                        team_id ) => {
-  //
+const constructBody = ( github,
+                        team_id,
+                        event_name,
+                        event_action,
+                        repo,
+                        environment,
+                        branch,
+                        commit,
+                        image,
+                        custom
+                        ) => {
+
   // The presence of any of these values below implies the user wants to specify
   // all of them. In which case, we won't use the GitHub Context object to
   // populate the event API body (we just return all existing values instead).
   //
-  if (change_id || stage_ref || custom || pipeline_id || stage || status) {
+  if (true || event_name || event_action || repo || environment || branch || commit || image) {
```

Comment:
why if true?

Reply:
To avoid running the automated payload detection for the time being, as it really complicates the logic, is messy and is an unnecessary feature right now since we now have the Github App.